proto/blueprint.proto

```diff
@@ -358,10 +364,13 @@ message Video {
   optional string orientation = 5;
   string url = 6;
   optional int32 width = 7;
-  optional string youtube_id = 8;
+  optional string youtube_id = 8 [deprecated = true];
```

Comment:
Looks like we do already have a precedent for this approach: 
https://github.com/guardian/mobile-apps-api-models/blob/40b9e43d2aa4ea0a3f9e3b3d395d9bb7c4772758/proto/blueprint.proto#L199 👍